site/Makefile

```diff
@@ -34,11 +34,14 @@ python_setup:
 	sudo pip install pdoc pygments
 
 setup: ruby_setup
+	scripts/html-setup.sh osx
+
+linkcheck_macos:
+	~/bin/htmltest-osx
```

Comment:
I'm not a big fun of putting the file under a home directory. Why don't you put it under scripts/ or tools/ ?